src/config.rs

```diff
@@ -3231,6 +3231,21 @@ impl ConfigController {
 
     pub fn update(&self, change: HashMap<String, String>) -> CfgResult<()> {
         let diff = to_config_change(change.clone())?;
+        self._update(diff, Some(change))
+    }
+
+    pub fn update_from_toml_file(&self) -> CfgResult<()> {
+        let current = self.get_current();
+        let incoming = TiKvConfig::from_file(Path::new(&current.cfg_path), None);
+        let diff = current.diff(&incoming);
+        self._update(diff, None)
+    }
+
+    fn _update(
```

Comment:
Prefer to name it `update_impl`. I think underscore prefixed function names is not commonly used in Rust

Reply:
Fixed, thanks!